embedded/app/main/badgerloop_LV.cpp

```diff
@@ -12,8 +12,8 @@ extern "C"
     #include "lv_iox.h"
     #include "nav.h"
     #include "braking.h"
-    
-#include "connStat.h"
+    extern void csvFormatShow();    
```

Comment:
This extern was more for debugging, and probably should be deleted from this file